engine/apps/mobile_app/migrations/0011_alter_mobileappusersettings_going_oncall_notification_timing.py

```diff
@@ -21,7 +21,11 @@ class Migration(migrations.Migration):
 
     operations = [
         linter.IgnoreMigration(),
-        migrations.AlterField(
+        migrations.RemoveField(
+            model_name='mobileappusersettings',
+            name='going_oncall_notification_timing',
+        ),
+        migrations.AddField(
```

Comment:
Had to rewrite this migration because PostgreSQL was complaining when trying to migrate previous int value to new json field:
```
psycopg2.errors.CannotCoerce: cannot cast type integer to jsonb
LINE 1: ..." TYPE jsonb USING "going_oncall_notification_timing"::jsonb
```